ecms-social-integration/src/main/java/org/exoplatform/wcm/ext/component/activity/FileUIActivity.java

```diff
@@ -883,6 +883,10 @@ public void setUIActivityData(Map<String, String> activityParams) {
     return (valuesArray == null || index > (valuesArray.length - 1)) ? null : valuesArray[index];
   }
 
+  private boolean isQuarantinedItem(Node node) throws RepositoryException {
+    return node.getPath().startsWith("/Security");
```

Comment:
Can you use FileDLPConnector.DLP_SECURITY_FOLDER ? 
As we will change this name in future, it iwll prevent us to forget this one :)

Reply:
Done!